ember/app/router.js

```diff
@@ -7,10 +7,13 @@ const Router = EmberRouter.extend({
   didTransition: function() {
     this._super(...arguments);
 
-    return ga('send', 'pageview', {
-      'page': this.get('url'),
-      'title': this.get('url')
-    });
+        // ga() undefined but should be from google analytics, should be able to add an import here
```

Comment:
I noticed there aren't issues enabled in the repository, this would probably be best addressed in a followup, let me know if you'd like to use issues.

Reply:
Feel free to enable issues for this repository.